draft-add-requirements.md

```diff
@@ -55,66 +92,119 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
-may require communication with the local resolver, or an upstream resolver, over an unencrypted transport. Once discovered, the local encrypted forwarder
-may be selected by the client, gaining the benefits of encryption while retaining the benefits of a local caching forwarder with knowledge of the local
-topology.
+If a subset of local resolvers supports encrypted DNS, the client may not initially be aware that its local resolver
+supports it. Discovering this may require communication with the local resolver, or an upstream resolver, over an
+unencrypted transport. Once discovered, the local encrypted forwarder may be selected by the client, gaining the
+benefits of encryption while retaining the benefits of a local caching forwarder with knowledge of the local topology.
 
 | Requirement | Description |
 | R4.1 | Example requirement |
 
-## Discover resolvers associated with a manually-configured resolver
+## Client-selected resolvers {#client-selected}
+
+Client devices often allow a user or administrator to select a specific DNS resolver to use
+on certain networks, or on all networks. Historically, this selection was specified only with an
+IP address.
+
+Discovering which equivalent encrypted resolvers are offered by the same entity allows the client
+to "upgrade" connections to use encrypted DNS.
+This can provide several benefits:
+
+- Prevent devices along the network path to the selected resolver from observing client DNS messages
+- Verify that answers come from the selected DNS resolver
+- Authenticate that the DNS resolver is the one selected by the client
+
+In doing so it is critical that the new resolver provides the same semantics as the original
+unencrypted one. Switching to one that provides different answers would break the expectation
+of the user who previously selected that resolver.
 
-A client with manual configuration directing it to use a particular resolver will generally use that resolver. However if that communication is
-unencrypted, it is beneficial for the client to learn which associated resolvers are offered by the same entity and provide the same semantics.
 
 | Requirement | Description |
 | R5.1 | Example requirement |
 
-## Discover resolvers associated with a VPN-configurated resolver
+## VPN resolvers {#vpn}
+
+Virtual Private Networks (VPNs) also can provision DNS resolvers. In addition to being able to use
+DHCP or RAs, VPNs can provision DNS information in an explicit configuration message. For example,
+IKEv2 can provision DNS servers using Configuration Attributes {{?RFC7296}}.
+
+VPNs can also configure Split DNS rules to limit the use of the configured resolvers to specific domain
+names {{?RFC8598}}.
+
+Discovering an encrypted resolver that is provisioned by a VPN can provide the same benefits
+as doing so for a local network, but applied to the private network. When using Split DNS, it becomes
+possible to use one encrypted resolver for private domains, and another for other domains.
 
```

Comment:
You can use the text from https://tools.ietf.org/html/draft-btw-add-ipsecme-ike-00#section-3 to discuss the use cases.

Reply:
Excellent; I had already planned to shamelessly copy your existing work (and the work found in other ADD drafts), but it's good to have the pointer.

Reply:
Sure, feel free to add text from existing drafts and you can update the Acknowledgements section accordingly 👍